backend/app.py

```diff
@@ -171,49 +170,167 @@ def jqQuery(objtype: str, query: str, objlist=None, ) -> Union[Dict, List]:
     return result
 
 
-# @APP.get("/fields-from-objlist")
-# def fields_from_objlist(
-#     objtype: str = Query(...),
-#     filter: Union[List[str], None] = Query(default=[]),
-#     field: Union[List[str], None] = Query(default=[]),
-# ) -> Union[Dict, List]:
-#     """
-#         get one or more fields from specified object type, example:
-#         http://127.0.0.1:8000/fields-from-objlist?field=concept_set_name&field=codeset_id&objtype=OMOPConceptSet
-#     """
-#
-#     queryClauses = []
-#     objlist = load_json(objtype)
-#     fields = validFieldList(objlist=objlist, fields=field)
-#     if len(fields):
-#         queryClauses.append('{' + ', '.join(fields) + '}')
-#
-#     valFilters = {k: v and v.split('|') or [] for k, v in [filt.split(':') for filt in filter]}
-#     filterFields = validFieldList(objlist=objlist, fields=valFilters.keys())
-#     for filterField in filterFields:
-#         filtVals = valFilters[filterField]
-#         if len(filtVals):
-#             condition = 'or'.join([f' .codeset_id == {val} ' for val in filtVals])
-#             clause = f'select({condition})'
-#             queryClauses.insert(0, clause)
-#
-#     queryClauses.insert(0, '.[]')
-#     query = ' | '.join(queryClauses)
-#     subset = jqQuery(objtype=objtype, objlist=objlist, query=query)
-#
-#     return subset
-#     # groupQuery = 'group_by(.concept_set_name) | map({ key: .[0].concept_set_name | tostring, value: [.[] | {version, codeset_id}] }) | from_entries'
-#     # res = jqQuery(objtype=objtype, objlist=subset, query=groupQuery)
-#     # return res
+# TODO: Finish this route
+# TODO: i. Fix: Very slow on large N. For one attempt, I got back ~7,000 results for 'concepts' variables. I think this is
+#  because one of the csets was 'immunotherapy'; a big set.
+# TODO: ii. Color table
+# Example: http://127.0.0.1:8000/concept-set-overlap-table-data?codeset_id=314083061|728628308|98365468
+@APP.get("/concept-set-overlap-table-data")
```

Comment:
Created placeholder route for: `/concept-set-overlap-table-data`. This will be for when we use the `concept_relationship_is_a.csv`

Reply:
This applies to all of the `/concept-set-overlap-table-data*` routes. There are ~3 variations now, though the front-end is only reading from `/concept-set-overlap-table-data-simple-hierarchy`
> `TODO: i. Fix: Very slow on large N. For one attempt, I got back ~7,000 results for 'concepts' variables. I think this is because one of the csets was 'immunotherapy'; a big set.`